### PR TITLE
Bem comp

### PR DESCRIPTION
**Pull request by StasDomanskyi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,49 +144,33 @@ <h2 class="title">Type of bicycles</h2>
...
+                    World Wars.
+                </p>
+                <p class="cycle-col__text">
+                    There are usually between 20 and 22 teams, with eight riders in each.
+                    All of the stages are timed to the finish; the riders' times are compounded
+                    with their previous stage times. The rider with the lowest cumulative
+                    finishing times is the leader of the race and wears the yellow jersey.
+                </p>
+                <a href="#" class="cycle-col__link">
+                    <button class="cycle-col__link_button">Read more</button>
```
Comment (LenaAviles) [inline index.html]: is `_button` modifier? then it should be separated by double dashes `--`

File: sass/variables.sass [hunk 2]
```diff
@@ -1,5 +1,37 @@
 $yellow: #ffdd57
 
+.spinner
```
Comment (LenaAviles) [inline sass/variables.sass]: why do you include spinner styles in file, named `variables.sass`?

File: sass/variables.sass [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+    width: 14rem
+    height: 14rem
+    background: rgb(49, 37, 32)
+    overflow: hidden
+    animation: move 2s infinite linear
+    border-radius: 50%
+    position: relative
+    box-shadow: 0 0 30px #222121b7
+
+.spinner__triangle
```
Comment (LenaAviles) [inline sass/variables.sass]: use sass syntax `&` instead of `.spinner`

File: style.css [hunk 4]
```diff
@@ -122,16 +122,43 @@
...
-    padding: 4rem;
+.cycle-col__text {
+    margin: 1rem;
+    padding: 0 1rem;
+    font-size: 1.2rem;
+    text-align: justify;
+}
+
+.cycle-col__link_button {
+    margin: 4rem auto 2rem;
```
Comment (LenaAviles) [inline style.css]: avoid using margin-top

File: index.html (hunk 1 above)
Comment (StasDomanskyi) [reply index.html]: okay, problem with name of class resolved

File: style.css (hunk 4 above)
Comment (StasDomanskyi) [reply style.css]: okay, we avoided it in this part of styles

**Pull request merged**